examples/co2_cm1106/src/main.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file main.cpp
+ * @author Antonio Vanegas @hpsaturn
+ * @date June 2018 - 2020
+ * @brief Particle meter sensor tests
+ * @license GPL3
+ * 
+ * Full documentation:
+ * https://github.com/kike-canaries/canairio_sensorlib#canairio-air-quality-sensors-library
+ * 
+ * Full implementation for WiFi and Bluetooth Air Quality fixed and mobile station:
+ * https://github.com/kike-canaries/canairio_firmware#canairio-firmware
+ * 
+ * CanAirIO project:
+ * https://canair.io
+ */
+
+#include <Arduino.h>
+#include <Sensors.hpp>
+
+void onSensorDataOk() {
+    Serial.print ("-->[MAIN] PM1: "+sensors.getStringPM1());
+    Serial.print (" PM2.5: " + sensors.getStringPM25());
+    Serial.print (" PM10: " + sensors.getStringPM10());
```

Comment:
because you only in this example, started it with cm1106 type, we dont need show the PM values. Please remove it, maybe it confused to users.

Reply:
done